core/controllers/base_test.py

```diff
@@ -339,25 +337,27 @@ def test_i18n_keys(self):
         master_key_list = self._extract_keys_from_json_file('en.json')
         self.assertGreater(len(master_key_list), 0)
 
+        supported_language_filenames = [
+            ('%s.json' % language_details['id'])
+            for language_details in feconf.SUPPORTED_SITE_LANGUAGES]
+
         filenames = os.listdir(
             os.path.join(os.getcwd(), self.get_static_asset_filepath(),
                          'assets', 'i18n'))
         for filename in filenames:
             if filename == 'en.json':
                 continue
 
-            self.log_line('Processing %s...' % filename)
```

Comment:
Hmm, it didn't really seem needed, and I'm not sure why it was there in the first place. If you look at the logs, the main info we want to track is the strings that don't exist for languages in the production site -- everything else is just noise. Do you see a use for it?